mattumland/Salon
# Feature/api calls

## Is this a fix or a feature?
+ Feature

## What is the change?
+ Refactors the App component to more succinctly render artPieces
+ Separates API calls from App component

## Where should the reviewer start?
+ src/components/App/App.js
+ src/api.js

## How should this be tested?
+ Run server, visit http://localhost:3000/
+ On load, 7 unique images should render on the landing page
+ No errors should be thrown from failed network requests
